design/help_overview.md

```diff
@@ -6,21 +6,21 @@ pywbemcli  --help
 =================
 
 ```
-b'Usage: pywbemcli [GENERAL-OPTIONS] COMMAND [ARGS]...\n\n  Command line browser for WBEM Servers. This cli tool implements the\n  CIM/XML client APIs as defined in pywbem to make requests to a WBEM\n  server.\n\n  The options shown above that can also be specified on any of the\n  (sub-)commands.\n\nOptions:\n  -s, --server TEXT               Hostname or IP address with scheme of the\n                                  WBEMServer (EnvVar: PYWBEMCLI_SERVER).\n  -d, --default_namespace TEXT    Default Namespace to use in the target\n                                  WBEMServer if no namespace is defined in the\n                                  subcommand(EnvVar:\n                                  PYWBEMCLI_DEFAULT_NAMESPACE). (Default:\n                                  root/cimv2).\n  -u, --user TEXT                 User name for the WBEM Server connection.\n                                  (EnvVar: PYWBEMCLI_USER).\n  -p, --password TEXT             Password for the WBEM Server (EnvVar:\n                                  PYWBEMCLI_PASSWORD ).\n  -t, --timeout TEXT              Operation timeout for the WBEM Server.\n                                  (EnvVar: PYWBEMCLI_TIMEOUT).\n  -n, --noverify                  If set, client does not verify server\n                                  certificate.\n  -x, --certfile TEXT             Server certfile. Ignored if noverify flag\n                                  set. (EnvVar: PYWBEMCLI_CERTFILE).\n  -k, --keyfile TEXT              Client private key file. (EnvVar:\n                                  PYWBEMCLI_KEYFILE).\n  -o, --output-format [mof|xml|table|csv|text]\n                                  Output format (Default: mof). pywbemcli may\n                                  override the format choice depending on the\n                                  operation since not all formats apply to all\n                                  output data types\n  --debug                         Display transmited and received packets\n                                  TEMP.\n  -v, --verbose                   Display extra information about the\n                                  processing.\n  --version                       Show the version of this command and exit.\n  --help                          Show this message and exit.\n\nCommands:\n  class       Command Group to manage CIM Classes.\n  connection  Command group to manage WBEM connections...\n  help        Show help message for interactive mode.\n  instance    Command Group to manage CIM instances.\n  qualifier   Command Group to manage CIM...\n  repl        Enter interactive (REPL) mode (default) and...\n  server      Command Group for WBEM server operations.\n'
```

Comment:
This results in a very long line, not only in the source file but also in the rendered output. Split accordingly.